components/ImprovedSearch/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import { Icon } from '@equinor/eds-core-react'
+import { chevron_left, chevron_right, first_page, last_page } from '@equinor/eds-icons'
+import { useContext, useEffect, useRef } from 'react'
+import { usePagination, UsePaginationProps } from 'react-instantsearch-hooks-web'
+import styled from 'styled-components'
+import { usePrefersReducedMotion } from './ReducedMotion'
+import { PaginationContext } from './PaginationContext'
+import { PaginationItem } from './PaginationItem'
+
+
+const PaginationList = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: grid;
+  grid-gap: 8px;
+  grid-auto-columns: min-content;
+  grid-auto-flow: column;
+`
+
+export type PaginationProps = {
+  hitsPerPage?: number
+  inverted?: boolean
```

Comment:
We don't need the inverted option. It's used by Envis because we use this component both on white and dark blue background, but we only have one background colour